roles/satellite/tasks/internals/activationkey-hostgroup-definition.yml

```
@@ -35,7 +35,17 @@
     parent: "{{ test_hostgroup_parent | default(omit) }}"
     pxe_loader: "{{ test_pxe_loader | default(omit) }}"
     locations: "{{ locations.keys() | list }}"
-    parameters: "{{ [ [ {'name' : 'kt_activation_keys', 'value': test_activationkey}], satellite_hostgroup_parameters_defaults, hostgroup_parameters | default([]), test_hostgroup_parameters | default([])] | community.general.lists_mergeby('name') }}"
+    parameters: >-
+      {{
+        (
+          [ {'name': 'kt_activation_keys', 'value': test_activationkey } ]
+          + (satellite_hostgroup_parameters_defaults | default([]))
```

I dont think default values also needs default values. 
```suggestion
          + (satellite_hostgroup_parameters_defaults )
```